sql/core/src/test/resources/sql-tests/results/pgSQL/with.sql.out

```diff
@@ -0,0 +1,486 @@
+-- !query 21 schema
+struct<>
+-- !query 21 output
+
+
+
+-- !query 22
+SELECT * FROM bug6051
+-- !query 22 schema
+struct<i:int>
+-- !query 22 output
+1
+2
+3
+
+
+-- !query 23
+CREATE TABLE bug6051_2 (i int) USING parquet
+-- !query 23 schema
+struct<>
+-- !query 23 output
+
+
+
+-- !query 24
+CREATE TABLE withz USING parquet AS SELECT i AS k, CAST(i || ' v' AS string) v FROM (SELECT EXPLODE(SEQUENCE(1, 16, 3)) i)
+-- !query 24 schema
+struct<>
+-- !query 24 output
+
+
+
+-- !query 25
+SELECT * FROM withz ORDER BY k
+-- !query 25 schema
+struct<k:int,v:string>
+-- !query 25 output
+1	1 v
+4	4 v
+7	7 v
+10	10 v
+13	13 v
+16	16 v
+
+
+-- !query 26
+DROP TABLE withz
+-- !query 26 schema
+struct<>
+-- !query 26 output
+
+
+
+-- !query 27
+TRUNCATE TABLE y
+-- !query 27 schema
+struct<>
+-- !query 27 output
+
+
+
+-- !query 28
+INSERT INTO y SELECT EXPLODE(SEQUENCE(1, 3))
+-- !query 28 schema
+struct<>
+-- !query 28 output
+
+
+
+-- !query 29
+CREATE TABLE yy (a INTEGER) USING parquet
+-- !query 29 schema
+struct<>
+-- !query 29 output
+
+
+
+-- !query 30
+SELECT * FROM y
+-- !query 30 schema
+struct<a:int>
+-- !query 30 output
+1
+2
+3
+
+
+-- !query 31
+SELECT * FROM yy
+-- !query 31 schema
+struct<a:int>
+-- !query 31 output
+
+
+
+-- !query 32
+SELECT * FROM y
+-- !query 32 schema
+struct<a:int>
+-- !query 32 output
+1
+2
+3
+
+
+-- !query 33
+SELECT * FROM yy
+-- !query 33 schema
+struct<a:int>
+-- !query 33 output
+
+
+
+-- !query 34
+CREATE TABLE parent ( id int, val string ) USING parquet
+-- !query 34 schema
+struct<>
+-- !query 34 output
+
+
+
+-- !query 35
+INSERT INTO parent VALUES ( 1, 'p1' )
+-- !query 35 schema
+struct<>
+-- !query 35 output
+
+
+
+-- !query 36
+SELECT * FROM parent
+-- !query 36 schema
+struct<id:int,val:string>
+-- !query 36 output
+1	p1
+
+
+-- !query 37
+SELECT * FROM parent
+-- !query 37 schema
+struct<id:int,val:string>
+-- !query 37 output
+1	p1
+
+
+-- !query 38
+create table foo (with baz)
+-- !query 38 schema
+struct<>
+-- !query 38 output
+org.apache.spark.sql.catalyst.parser.ParseException
+
+DataType baz is not supported.(line 1, pos 23)
+
+== SQL ==
+create table foo (with baz)
+-----------------------^^^
+
+
+-- !query 39
+-- fail, WITH is a reserved word
+create table foo (with ordinality)
+-- !query 39 schema
+struct<>
+-- !query 39 output
+org.apache.spark.sql.catalyst.parser.ParseException
+
+DataType ordinality is not supported.(line 2, pos 23)
+
+== SQL ==
+-- fail, WITH is a reserved word
+create table foo (with ordinality)
+-----------------------^^^
+
+
+-- !query 40
+-- fail, WITH is a reserved word
+with ordinality as (select 1 as x) select * from ordinality
+-- !query 40 schema
+struct<x:int>
+-- !query 40 output
+1
+
+
+-- !query 41
+WITH test AS (SELECT 42) INSERT INTO test VALUES (1)
+-- !query 41 schema
+struct<>
+-- !query 41 output
+org.apache.spark.sql.AnalysisException
+unresolved operator 'InsertIntoTable 'UnresolvedRelation [test], false, false;
```

Comment:
Originally, PostgreSQL shows `ERROR:  relation "test" does not exist`.
The current Spark's error message looks weird in this case. Could you file an Spark JIRA issue for this, please?

Reply:
Yes, I created https://issues.apache.org/jira/browse/SPARK-28251

Reply:
Yep. I merged that. Thanks.